gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -5,12 +5,16 @@ import org.gradle.api.Project
 
 public class AndroidCommandPluginExtension {
 
+    static final String CATEGORIES_DEFAULT = ''
```

Comment:
but doing a blank -c allows the adb command to use the default categories (the same as not setting it)